js/5etools-main.js

```diff
@@ -866,6 +866,10 @@ const betteR205etoolsMain = function () {
 		}
 
 		const $body = $("body");
+
+		// Text for force external request warning
+		const ferWarning = `<div><b style="color:red">WARNING: You are currently force fetching data from an external source, rather than the script's inbuilt data. This takes more time and is more demanding on the site. Do not use this feature unless necessary.</b></div>`
```

Comment:
Reword as:
```
WARNING: You are currently fetching data from an external source, rather than the script's inbuilt data. External data may not be compatible with your current script version. Do not use this feature unless necessary.
```
since this is the meaningful downside for the user